test/metabase/lib/fe_util_test.cljc

```diff
@@ -201,7 +201,8 @@
   (let [query         (lib.tu/venues-query)
         column        (meta/field-metadata :venues :price)
         bigint-string "9007199254740993"
-        bigint-value  (u.number/parse-bigint bigint-string)]
+        bigint-value  (u.number/parse-bigint bigint-string)
+        bigint-clause (lib/expression-clause :value [bigint-string] {:base-type :type/BigInteger :effective-type :type/BigInteger})]
```

Comment:
I like this phrasing better than the `ensure-uuid` call I flagged in `fe-util`.

Perhaps this should go in `lib.expression` with all the other `lib/+` etc. as a helper, like `(lib/bigint bigint-or-string)`